cell_classification/configs/params.toml

```diff
@@ -0,0 +1,32 @@
+record_path = "C:/Users/lorenz/Desktop/angelo_lab/MIBI_test/TNBC_CD45.tfrecord"
+path = "C:/Users/lorenz/OneDrive/Desktop/angelo_lab/"
+experiment = "test"
+num_epochs = 10
+lr = 1e-3
+backbone = "resnet50"
+input_shape = [256,256,2]
+num_validation = 500
+shuffle_buffer_size = 2000
+flip_prob = 0.5
+affine_prob = 0.5
+scale_min = 0.8
+scale_max = 1.2
+shear_angle = 0.2
+elastic_prob = 0.5
+elastic_alpha = 100
+elastic_sigma = 5
+rotate_count = 4
+gaussian_noise_prob = 0.5
+gaussian_noise_min = 0.05
+gaussian_noise_max = 0.15
+gaussian_blur_prob = 0.5
+gaussian_blur_min = 0.05
+gaussian_blur_max = 0.15
+contrast_prob = 0.5
+contrast_min = 0.8
+contrast_max = 1.2
+raw_keys = ["mplex_img"]
+label_keys = ["binary_mask", "marker_activity_mask"]
+batch_size = 4
+[classes]
```

Comment:
What's this for?

Reply:
Deepcells `PanopticNet` expects a dictionary for the argument `num_semantic_classes` that maps output keys to output channels, i.e. {"marker_positive":1} which is what params["classes"] when it's written like this in the toml.